components/breadcrumb/stories/breadcrumb.stories.js

```diff
@@ -59,23 +104,191 @@ Default.args = {
 	],
 };
 
-export const NestedMultiline = Template.bind({});
-NestedMultiline.args = {
+export const DefaultNested = Template.bind({});
+DefaultNested.args = {
+	items: [
+		{
+			iconName: "FolderOpen",
+			iconSet: "workflow",
+		},
+		{
+			label: "Sub item",
+		},
+		{
+			label: "Trend",
+		},
+		{
+			label: "January 2019 assets",
+		},
+	],
+};
+DefaultNested.tags = ["!dev"];
```

Comment:
Noting that all the stories are showing in the sidebar since this branch doesn't know what `"!dev"` is, but I think that's fine.